ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLandOwnership.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <v-card flat rounded id="mhr-home-land-ownership" class=" mhr-home-land-ownership">
+    <v-row no-gutters>
+      <v-col cols="12" sm="2">
+        <label class="generic-label" for="ownership">
+        Land Lease or Ownership
+        </label>
+      </v-col>
+      <v-col cols="12" sm="10">
+        <v-checkbox
+          id="ownership"
+          label="The manufactured home is located on land that the homeowners own,
+                or on which they have a registered lease of 3 years or more."
+          v-model="isOwnLand"
+          class="my-0 py-0 px-0 ownership-checkbox"
+          data-test-id="ownership-checkbox"
+        />
+       </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
+import { useGetters, useActions } from 'vuex-composition-helpers'
+
+export default defineComponent({
+  name: 'HomeLandOwnership',
+  setup () {
+    const {
+      getMhrRegistrationOwnLand
+    } = useGetters<any>([
+      'getMhrRegistrationOwnLand'
+    ])
+
+    const {
+      setMhrRegistrationOwnLand
+    } = useActions<any>([
+      'setMhrRegistrationOwnLand'
+    ])
+
+    const localState = reactive({
+      isOwnLand: Boolean(getMhrRegistrationOwnLand.value || false)
```

Comment:
is `Boolean()` required here? 

The computed return value is defined as boolean, if TS has an issue on this ref, you could also `!!getMhrRegistrationOwnLand.value..`. 

Either way, should try and stick to primitive data types if possible, Boolean and boolean are (oddly) different, the latter is primitive while Boolean is reserved for object references, which i don't believe `getMhrRegistrationOwnLand.value` is.

Reply:
Good point ill change that to just

`isOwnLand: !!getMhrRegistrationOwnLand.value`